vars/shared-OpenData.var.yml

```diff
@@ -184,6 +193,10 @@ extensions:
       <<: *CKANExtDataRequests
       version: "develop"
 
+    CKANExtDataQldHarvest:
+      <<: *CKANExtDataQldHarvest
+      version: "develop"
```

Comment:
There is no `develop` branch.

Reply:
there is  https://github.com/qld-gov-au/ckanext-data-qld-harvester/tree/develop

Reply:
ps this is not https://github.com/ckan/ckanext-harvest this is not 'forked'